src/components/Chat/ChatDetails.tsx

```diff
@@ -163,30 +162,40 @@ export class ChatDetails extends React.PureComponent<ChatDetailsProperties, Chat
                             <i className="fa fa-trophy" /> {tournament_text}
                         </button>
                     )}
+                    <hr />
                     {this.state.subscribable && (
-                        <button
-                            className={"xs noshadow "} // + this.state.notify_mentioned ? "active" : "inactive"}
-                            onClick={this.toggleMentionNotification}
-                        >
-                            <i className="fa fa-comment" />
-                            {" " +
-                                (this.state.notify_mentioned
-                                    ? _("unfollow mentioned")
-                                    : _("follow mentioned"))}
-                        </button>
+                        <div>
+                            <input
+                                type="checkbox"
+                                id="notify_mentioned"
+                                checked={this.state.notify_mentioned}
+                                onChange={this.toggleMentionNotification}
+                            />
+                            <label htmlFor="notify_mentioned">
+                                {pgettext(
+                                    "Don't notify on unread @user mentions in a chat channel",
```

Comment:
Nit: I think having the hint be the opposite (don't notify) of the actual text (do notify) could lead to confusion for translators.